amundsen_application/static/js/components/TableDetail/ColumnStats/index.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as moment from 'moment-timezone';
+
+import { TableColumnStats } from 'interfaces/index';
+
+import './styles.scss';
+
+export interface ColumnStatsProps {
+  stats: TableColumnStats[];
+}
+
+export class ColumnStats extends React.Component<ColumnStatsProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  formatDate = (unixEpochSeconds) => {
+    return moment(unixEpochSeconds * 1000).format("MMM DD, YYYY");
+  };
+
+  getStatsInfoText = (startEpoch: number, endEpoch: number) => {
+    const startDate = startEpoch ? this.formatDate(startEpoch) : null;
+    const endDate = endEpoch ? this.formatDate(endEpoch) : null;
+
+    let infoText = 'Stats reflect data collected';
+    if (startDate && endDate) {
+      if (startDate === endDate) {
+        infoText = `${infoText} on ${startDate} only. (daily partition)`;
+      } else {
+        infoText = `${infoText} between ${startDate} and ${endDate}.`;
+      }
+    } else {
+      infoText = `${infoText} over a recent period of time.`;
```

Comment:
We can delete this case if you want. It protects against a case that shouldn't happen -- not having `startEpoch` or `endEpoch`. My thinking here is that maybe we don't want to hide this issue in the UI so that users will actually recognize it as a bug if it ever happens.

Reply:
Should we show an error message, an empty string, or actually throw an error?

Reply:
Unsure. 🤔I think about how this is probably a bigger issue of validating -- somewhere earlier in this process -- that the data we receive from API calls match what the application expects. I don't think it should be the responsibility of the component to account for things that violate the TypeScript definition, rather some earlier step in ducks or the Flask layer. We can table this for the future.